aur.go

```diff
@@ -27,7 +27,7 @@ type response struct {
 type By int
 
 const (
-	Name By = iota
+	Name By = iota + 1
```

Comment:
What's the benefit of changing this? If you need it for the test can't you test for something that is > the max?

Reply:
The default value is zero. This means that sometimes, due to an error, zero may get into the program and main program will work incorrectly.

Reply:
In short it is "foolproof"

Reply:
OK, I guess it's not a problem to change as it doesn't change the API (if it's being used in the right way).